### ISSUE TITLE
Website v0.1

### ISSUE THREAD

**Issue by rufuspollock** [life-itself/web3#66]
Launchable and *announceable* website.

Sketch of home page
![website-homepage-v0.1-2022-02-23.excalidraw](https://raw.githubusercontent.com/life-itself/web3/2f738dd8b62196b285a760e34ff9847cf7c0caaa/excalidraw/website-homepage-v0.1-2022-02-23.excalidraw.svg)

Mockup of home page (design): https://excalidraw.com/#room=8bc8a0afb0afc89015dc,W3jd48r8_B2llA2CZ8A33A

## Acceptance

* [x] Home Page
  * [x] ⏫ Looks reasonable
  * [x] Hero section
  * [x] 🔼 Latest 2-3 videos up somewhere on the front page
  * [x] Guide section
  * [x] Library section
  * [x] About the project
  * [x] Contribute @EilidhRoss1 can you finish adding this ✅ **Get Involved Section added** 
* [x] About page **✅ with SCQH**
* [x] Library page
* [x] ⏫ Sign up to get updates / news …
* [x] Misc
  * [x] ⏫ Analytics
  * [x] Navbar
  * [x] Social media image **✅ used front page screenshot**
  * [x] Footer
    * [x] Social media links
    * [x] License info

Bonus

* [x] Guide page

## Tasks

Overview

* [x] Home page - see below
* [x] Library page **✅ https://web3.lifeitself.us/library**
* [x] Guide page **✅ Have a more than decent guide page https://web3.lifeitself.us/guide**
* [x] Notes section (wiki) **Have notes showing up on the site. No index page though**

### Home Page

* [x] Hero
  * [x] Layout
  * [x] Fix "get updates"
    * [x] EITHER: Remove "Get updates" & replace with CTA OR keep Get Updates using netlify form support => Choose what to do **✅ trying out netlify forms https://docs.netlify.com/forms/setup/ docs**
    * [x] Do it **✅2022-03-14 trying it out and seems to be working**
* [x] Latest releases
  * [x] Working
  * [x] Replace images either with YouTube video embed or the thumbnail image (?) @eilidh
    * [x] Upload images to cloudinary
    * [x] Add to Latest Release section
* [x] Guide section
* [x] Library section
  * [x] Image needs changing. **✅ DONE [new image link](https://res.cloudinary.com/ds7qslkd0/image/upload/v1646739680/Web3/undraw_bibliophile_re_xarc_ssxnqe.svg )**
* [x] Why the project
* [x] Get involved (front page) ✅ **Get Involved Section added** 

### Library Page

Acceptance

* [x] `/library` landing page exists
* [x] brief overview
* [x] links to zotero
* [x] Also explanation of how to contribute incl organizational structure **🏃<U+200D>♂️ Done the first part and need to migrate the rest of the library contribute guide doc @eilidhross1**

Tasks

* [x] Basic text for library page e.g. what, etc **✅ Eilidh written. Can be found [here](https://docs.google.com/document/d/1CtQ5KsdicM_d4oI-Px7DZRls6OwxeBozVO3vgiXw7m8/edit#)**
* [x] Link or embed the full library

### Guide

plan of content https://coggle.it/diagram/YidoEewvuAnxT9LA/t/web3-crypto-guide-toc

* [x] Stub `/guide/` page and could reuse quite a bit of about as introduction

### Contribute `/contribute/`

* [x] Developers => contribute on github
* [x] Non-developers
  * [x] Researchers / academics etc
* [x] General: check out our videos and resources and Share the work with others
* [x] Feedback: share it on the discussions

### Inbox

* [ ] Logo and favicon (could auto-generate)
* [x] Issue tree image
  * [x] Export it or embed? **Export image**
  * [x] Make it simpler
* [x] can we use symlinks to display content from main part of the repo into the site itself **✅ Yes you can. Rufus has this working**

**Comment by rufuspollock** [comment]
Almost done on this.

**Comment by rufuspollock** [comment]
@EilidhRoss1 have you managed to get the "get involved" section done?

**Comment by EilidhRoss1** [comment]
@rufuspollock Not yet, Khalil and I had our call on Thursday. I recorded the conversation and I'm hoping to begin creating the editors' guide from that this afternoon. My plan is to do the get involved section as I follow along with the tutorial I recorded with Khalil, so as to make sure that the tutorial is successful in demonstrating how to edit the website. So I should have this done by tomorrow.
